Snakefile

```diff
@@ -290,6 +290,24 @@ rule build_bus_regions:
 
 
 if config["enable"].get("build_cutout", False):
+    for technology in cutouts_in_config:
+        cutout_fl = (
+            "cutouts/" + CDIR + config["renewable"][technology]["cutout"] + ".nc"
+        )
+        if os.path.exists(cutout_fl):
+            raise Exception(
+                "A cutout file '"
+                + cutout_fl
+                + "' still exists. Stopping to avoid over-writing it"
+            )
+
+    cutout_fl = "cutouts/" + CDIR + list(config["atlite"]["cutouts"].keys())[0] + ".nc"
+    if os.path.exists(cutout_fl):
+        raise Exception(
+            "A cutout file '"
+            + cutout_fl
+            + "' still exists. Stopping to avoid over-writing it"
+        )
```

Comment:
Hello @ekatef and amazing feature :D
I feel like tlines from 304 to 310 are not needed from a functionality perspective.
To make it more general, we may fuse the two blocks into something like:
`
```
cutout_names = [d_value["cutout"]for tech, d_value in config["renewable"]] + list(config["atlite"]["cutouts"].keys())
for c_name in cutout_names:
       cutout_fl = "cutouts/" + CDIR + c_name + ".nc"
       ...
```